src/index.js

```diff
@@ -0,0 +1,149 @@
+    listItem.className = 'list-item';
+    listItem.innerHTML = `<input class="list-input" type="checkbox"><textarea name="textarea cols="30" class="item-details">${description}</textarea><i class="fas fa-ellipsis-v"></i>`;
+    list.appendChild(listItem);
+  }
+  const listInput = document.querySelectorAll('.list-input');
+  interact(listInput);
+  clearList(listInput);
+  removeItem();
+  updateValues();
+};
+
+const form = document.getElementById('form-id');
+const showTasks = () => {
+  form.addEventListener('submit', (e) => {
+    e.preventDefault();
+    const task = form.elements[0].value;
+
+    const data = {
+      description: task,
+      completed: false,
+      index: todoArr.length + 1,
+    };
+    form.reset();
+    todoArr.push(data);
+    localStorage.setItem('storeLocal', JSON.stringify(todoArr));
+    addTodoItem();
+  });
+};
+
+const removeItem = () => {
+  const button = document.querySelectorAll('.fa-ellipsis-v');
+  button.forEach((item) => {
+    const parent = item.parentNode;
+    const superParent = parent.parentNode;
+    const index = Array.prototype.indexOf.call(superParent.children, parent);
+    const listInput = parent.firstChild;
+    item.addEventListener('click', () => {
+      const storeLocal = JSON.parse(localStorage.getItem('storeLocal'));
+      todoArr.splice(0, todoArr.length, ...storeLocal);
+      if (listInput.hasAttribute('checked')) {
+        parent.remove();
+        todoArr.splice(index, 1);
+      }
+      for (let i = 0; i < todoArr.length; i += 1) {
+        todoArr[i].index = i + 1;
+      }
+      localStorage.setItem('storeLocal', JSON.stringify(todoArr));
+      addTodoItem();
+    });
+  });
+};
+
+const clearList = () => {
+  const getClearElement = document.querySelector('.clear-task');
+  getClearElement.addEventListener('click', () => {
+    for (let i = 0; i < todoArr.length; i += 1) {
+      if (todoArr[i].completed) {
+        todoArr.splice(i, 1);
+      }
+    }
+    for (let i = 0; i < todoArr.length; i += 1) {
+      todoArr[i].index = i + 1;
+    }
+    localStorage.setItem('storeLocal', JSON.stringify(todoArr));
+    addTodoItem();
+  });
+};
+
+const updateValues = () => {
+  const itemDetails = document.querySelectorAll('item-details');
+  itemDetails.forEach((item) => {
+    const parent = item.parentNode;
+    const superParent = parent.parentNode;
+    const index = Array.prototype.indexOf.call(superParent.children, parent);
+    item.addEventListener('change', () => {
+      todoArr[index].description = item.value;
+      localStorage.setItem('storeLocal', JSON.stringify(todoArr));
+    });
+  });
+};
+
+const adJusttext = (listInput) => {
+  listInput.forEach((item) => {
+    if (item.hasAttribute('checked')) {
+      item.nextSibling.style.textDecoration = 'line-through';
+    } else {
+      item.nextSibling.style.textDecoration = 'none';
+    }
+  });
+};
+
+const litstorage = () => {
+  window.addEventListener('load', () => {
+    addTodoItem();
```

Comment:
- [ ] Your to-do tasks are not rendered when the page loads even if your `todoArr` array has some to-do task inside. 🔨 Please, It would be better to implement a function to render your to-do tasks when the page loads.

![image](https://user-images.githubusercontent.com/79876959/145334222-0d2a0070-2b16-4949-8100-afe23c00008d.png)